frontend/src/metabase-types/api/card.ts

```diff
@@ -182,11 +182,46 @@ export type CardFilterOption =
   | "using_model"
   | "archived";
 
-export interface CardQuery {
+export interface ListCardsRequest {
+  f?: CardFilterOption;
+  model_id?: CardId;
+}
+
+export interface GetCardRequest {
+  id: CardId;
   ignore_view?: boolean;
 }
 
-export interface CardListQuery {
-  f?: CardFilterOption;
-  model_id?: CardId;
+export interface CreateCardRequest {
+  name: string;
+  dataset_query: DatasetQuery;
```

Comment:
do we duplicate Card type here?

Reply:
It's not the same as `Card` - different fields

Reply:
I could use something like `Pick<Card, "id"> & Pick<Partial<Card>, "name" | ...>`

Reply:
Wdyt?

Reply:
no, not necessary atm, maybe we'll come up to some better type in the future. We have differences between FE and API fields and it's better to have fields duplicated I'd say